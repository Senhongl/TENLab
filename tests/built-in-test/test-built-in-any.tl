a = [0,1,2];
if (any(a)){
    print("any-built-in functions works!");
}
b = [0,0,0];
if (any(b)){
    print("any-built-in functions doesn't work");
}