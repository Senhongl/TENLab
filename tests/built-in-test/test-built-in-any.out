any-built-in functions works!